frontend/src/components/nodes/NodeNav.tsx

```diff
@@ -66,21 +66,66 @@ export default class extends React.Component<Props, State> {
               </Col>
             </a>
           </Link>
-          <Link href="/nodes/[role]" as={`/nodes/non-validators`}>
+          <Link href="/nodes/[role]" as={`/nodes/online-nodes`}>
             <a
               className={`node-link ${
-                role === "non-validators" ? `node-selected` : ""
+                role === "online-nodes" ? `node-selected` : ""
               }`}
               id="non-validator-node"
             >
               <Col className="node-selector align-self-center">
                 {nodeStats
-                  ? `${nodeStats.nonValidatorsCount}  Non-Validating`
-                  : `- Non-Validating`}
+                  ? `${nodeStats.onlineNodesCount} Online-nodes`
+                  : `- Online-nodes`}
               </Col>
             </a>
           </Link>
+          <Link href="/nodes/[role]" as={`/nodes/proposals`}>
+            <a
+              className={`node-link ${
+                role === "proposals" ? `node-selected` : ""
+              }`}
+              id="proposal-node"
+            >
+              <Col className="node-selector align-self-center">
+                {nodeStats
+                  ? `${nodeStats.proposalsCount} Proposal-nodes`
+                  : `- Proposal-nodes`}
+              </Col>
+            </a>
+          </Link>
+        </Row>
+        <Row className="node-icon">
```

Comment:
I suggest we just add these hints inline, so the user can just hover over the circle on the left to the node id and read the hint.